recidiviz/ingest/us_vt/us_vt_scraper.py

```
@@ -0,0 +1,614 @@
+                bond_amount=charge['BondAmount'],
+                bond_type=charge['BondType'],
+                number_of_counts=charge['Counts'],
+                arresting_agency=charge['ArrestingAgency'],
+                court_type=charge['CourtType'],
+                warrant_number=charge['WarrantNumber'],
+                court_time=scraper_utils.parse_date_string(
+                    charge['CourtTime'], arrest_no),
+                control_number=charge['ControlNumber'],
+                crime_type=charge['CrimeType'],
+                arrest_code=charge['ArrestCode'],
+                modifier=charge['Modifier'],
+                arrest_date=scraper_utils.parse_date_string(
+                    charge['ArrestDate'], arrest_no),
+                )
+            record_offenses.append(offense)
+        record.offenses = record_offenses
+
+        record.status = person['Status']
+        record.release_date = roster_entry['FinalReleaseDateTime']
+        record.earliest_release_date = scraper_utils.parse_date_string(
+            person['Min Release'], arrest_no)
+        record.latest_release_date = scraper_utils.parse_date_string(
+            person['Max Release'], arrest_no)
+        record.parole_officer = person['Parole Officer']
+        record.case_worker = person['Case Worker']
+
+        record.sex = inmate.sex
+        record.race = inmate.race
+        record.last_name = inmate.last_name
+        record.given_names = inmate.given_names
+        record.record_id = str(roster_entry['ArrestNo'])
+
+        # guess facility vs. community supervision agency by looking
+        # for the string 'parole' in the agency name, and taking the
+        # first entry without it.
+        #
+        # this is a hacky method that should be verified.
+        facilities = [agency for agency in agencies
+                      if 'parole' not in agency.lower().rstrip()]
+        if not facilities.empty():
+            record.latest_facility = facilities[0]
```

Is it possible for a Vermont record to have multiple incarceration facilities listed?

---

AFAIK there will only be one facility and/or one supervision agency. I added a clarifying comment.

---

I'm reconsidering this. Perhaps in cases like this where we know assumptions are being made that will silently discard data if not met (more than one facility, for example), we should test for those conditions explicitly and fail so we catch it (here test that there is no more than a single entry in the facilities or cs_facilities arrays). Thoughts?